index.bs

```diff
@@ -2352,6 +2357,11 @@ To <dfn>compute the channel capacity of a source</dfn> given a [=randomized resp
 
 Note: This algorithm computes the channel capacity [[CHAN]] of a q-ary symmetric channel [[Q-SC]].
 
+To <dfn>compute the information gain for attribution scope</dfn> given an integer |numTriggerStates|, a positive integer |attributionScopeLimit| and a positive integer |maxEventStates|:
+1. Let |totalStates| be |numTriggerStates| + (|attributionScopeLimit| - 1) * |maxEventStates|.
```

Comment:
In the header validator the `maxEventStates` factor is on the left side of the multiplication. Let's be consistent for readability.